src/pages/en/guides/integrations-guide.md

```diff
@@ -44,6 +46,34 @@ If you see an error on startup, make sure that you:
 - ✅ called your integration as a function (`[react()]`, not `[react]`)
 - ✅ removed the deprecated `renderers:` configuration
 
+## Automatic Integration Setup
+
+Astro recent launched an **experimental** `astro add` command to automate the setup of integrations.
+
+> We will always ask for confirmation before updating any of your files, but it never hurts to have a version controlled backup just in case.
```

Comment:
```suggestion
> We will always ask for confirmation before updating any of your files, but it never hurts to have a version-controlled backup just in case.
```